ext/gd/gd.c

```diff
@@ -2738,6 +2783,16 @@ PHP_FUNCTION(imagegd2)
 }
 /* }}} */
 
+#ifdef HAVE_GD_BMP
+/* {{{ proto bool imagebmp(resource im [, string filename [, int compression]])
```

Comment:
Good catch! Indeed $compression is [treated as bool](https://github.com/php/php-src/pull/2029/files/ae3aae08701dfba9963448a6b7eeee91a9a3f9e4#diff-885d229df6b700a9fa716425290cb728R196), but actually imagebmp() accepts [an int](https://github.com/php/php-src/pull/2029/files/ae3aae08701dfba9963448a6b7eeee91a9a3f9e4#diff-885d229df6b700a9fa716425290cb728R102). Wrt. strict_types probably _php_image_output_ctx() has to be changed to have an own ZPP for PHP_GDIMG_TYPE_BMP.